docs/production/upgrade.md

```diff
@@ -686,16 +688,17 @@ instructions for other supported platforms.
    /home/zulip/deployments/current/manage.py audit_fts_indexes
    ```
 
-9. [Upgrading from Debian 10 to 11](#upgrading-from-debian-10-to-11),
-   so that you are running a supported operating system.
+9. [Upgrade from Debian 10 to 11](#upgrading-from-debian-10-to-11),
+   the next in chain of upgrades leading to a supported operating
+   system.
 
 ## Upgrading PostgreSQL
 
 Starting with Zulip 3.0, we use the latest available version of
 PostgreSQL at installation time (currently version 15). Upgrades to
```

Comment:
Is that still Postgres 15, or did we fail to update this when adding 16 support?